src/coreclr/vm/prestub.cpp

```diff
@@ -1608,12 +1611,6 @@ Stub * CreateInstantiatingILStub(MethodDesc* pTargetMD, void* pHiddenArg)
     // InstantiatingStub
     pCode->EmitLDC((TADDR)pHiddenArg);
 
-    // Push the async continuation
-    if (msig.HasAsyncContinuation())
-    {
-        pCode->EmitLDNULL();
-    }
```

Comment:
This is probably not dead code since GVMs may be called via instantiating stubs. I'll see if I can add a test and return this. We probably need similar handling for unboxing stubs, opened #117266 for that.

Reply:
I also looked at the use of `GetParamTypeArgOffset` and there is a managed version used in crossgen when generating pointer maps for calls. I suspect we may need something similar for async.